src/mockServiceWorker.js

```diff
@@ -181,7 +181,9 @@ If you wish to mock an error response, please refer to this guide: https://redd.
 function serializeHeaders(headers) {
   const reqHeaders = {}
   headers.forEach((value, name) => {
-    reqHeaders[name] = value
+    reqHeaders[name] = reqHeaders[name]
+      ? [].concat(reqHeaders[name]).concat(value)
```

Comment:
Does this need to join the headers in a list?

Reply:
This is fine, since the worker serializes the `request.headers` in order to send them to the client. On the client's side, the serialized instance is used to create a polyfilled `new Headers(request.headers)` instance, which supports `Record<string, string | string[]>` as an input, and joins a header's multiple values with a comma.